lib/types.js

```diff
@@ -92,7 +92,7 @@
  */
 
 /**
- * One of `main`, `testnet`, `regtest`, `segnet3`, `segnet4`.
+ * One of `main`, `testnet`, `regtest`, `simnet`, `segnet4`.
```

Comment:
This isn't even completely accurate: https://github.com/bcoin-org/bcoin/blob/master/lib/protocol/networks.js#L33

Reply:
I see, should I also remove [this](https://github.com/bcoin-org/bcoin/blob/master/lib/protocol/network.js#L376) reference to segnet4 then?

Reply:
good catch, sure!

Reply:
Done! I also found some more references in [scripts/gen.js](https://github.com/bcoin-org/bcoin/blob/master/scripts/gen.js#L86) and removed them.